api/models/userDao.js

```diff
@@ -36,4 +36,19 @@ const getUserBySocialId = async(socialId) => {
     return userInfo
 }
 
-module.exports = { getUserBySocialId, createUser }
\ No newline at end of file
+const getUserById = async(userId) => {
+    const result = await appDataSource.query(
+        `SELECT
+            id,
+            name,
+            email,
+            gender,
+            social_id
+        FROM users
+        WHERE id=?
+        `, [userId]
+    )
+    return result[0]
```

Comment:
다음과 같이 수정되면 좋을 것 같습니다.
```suggestion
    const [result] = await appDataSource.query(
        `SELECT
            id,
            name,
            email,
            gender,
            social_id
        FROM users
        WHERE id=?
        `, [userId]
    )
    return result
```